setup/src/Magento/Setup/Test/Unit/Console/Command/MaintenanceAllowIpsCommandTest.php

```diff
@@ -99,4 +126,31 @@ public function executeDataProvider()
             ]
         ];
     }
+
+    /**
+     * return array
+     */
+    public function executeWithAddDataProvider()
+    {
+        return [
+            [
+                [],
+                ['ip' => ['127.0.0.1'], '--add' => true],
+                [],
+                'Set exempt IP-addresses: 127.0.0.1' . PHP_EOL,
+            ],
+            [
+                ['127.0.0.1'],
+                ['ip' => ['127.0.0.1'], '--add' => true],
+                [],
+                'Set exempt IP-addresses: 127.0.0.1' . PHP_EOL,
+            ],
+            [
+                ['127.0.0.1'],
+                ['ip' => ['127.0.0.2'], '--add' => true],
+                [],
+                'Set exempt IP-addresses: 127.0.0.1, 127.0.0.2' . PHP_EOL,
```

Comment:
Can you please remove the comma sign between IP-addresses on the line? [#L152](https://github.com/barryvdh/magento2/blob/5747bb38ee4c64c7c6732cdd02a2abf5ac68a349/setup/src/Magento/Setup/Test/Unit/Console/Command/MaintenanceAllowIpsCommandTest.php#L152). I believe that this will help.

The test fails with:
```
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'Set exempt IP-addresses: 127.0.0.1, 127.0.0.2\n
+'Set exempt IP-addresses: 127.0.0.1 127.0.0.2\n
```

Please, fix the test and push again.

Reply:
Thanks, that was changed since I made this PR. I've updated it now.